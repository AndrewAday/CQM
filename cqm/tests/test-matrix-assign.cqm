int main()
{
  fmatrix fm;
  int i;
  int j;

  fm = init_fmat_const(2.45, 3, 5);

  fm[1,1] = 1.34;
  for (i = 0; i < rows(fm); i = i + 1) {
    for (j = 0; j < cols(fm); j = j + 1) {
      print_float(fm[i,j]);
      if (j == cols(fm) - 1) {
        print_line();
      }
    }
  }

  print_mat(fm);

  return 0;
}
