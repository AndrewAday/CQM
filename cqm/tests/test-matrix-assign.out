2.450000
2.450000
2.450000
2.450000
2.450000

2.450000
1.340000
2.450000
2.450000
2.450000

2.450000
2.450000
2.450000
2.450000
2.450000

2.45 2.45 2.45 2.45 2.45
2.45 1.34 2.45 2.45 2.45
2.45 2.45 2.45 2.45 2.45
